manifestProvider.go

```diff
@@ -0,0 +1,30 @@
+package onfleet
+
+type TurnByTurn struct {
+	StartAddress    string   `json:"startAddress"`
+	EndAddress      string   `json:"endAddress"`
+	ETA             int64    `json:"eta"`
```

Comment:
all caps `ETA` isn't consistent with other parts of the wrapper. e.g. `task.go`
```
	Eta                      *int64                   `json:"eta"`
```
I don't mind `ETA` (maybe it's better even) but i think if we use `ETA` here we should change all other relevant files to match that

Reply:
Alright, changed the code to maintain consistency